fixd.js

```diff
@@ -1,14 +1,30 @@
-const fs = require('fs')
-const path = require('path')
-const fixd = require('./src/index')
+'use strict'
+const merge = require('lodash.merge')
 
-// Auto-load built-in plugins
-try {
-  fs.readdirSync(path.resolve(__dirname, './src/plugins')).forEach((f) => {
-    fixd.use(f.replace('.js', ''))
-  })
-} catch (e) {
-  throw new Error('Error loading built-in plugins', e.message)
+const fixd = {
+  /**
+   * Add a fixture to the fixd library and freezes if not in namespace
+   * @param {String} name A string name to represent the fixture
+   * @param {*} value The fixture value
+   */
+  add: (name, val) => {
+    if (Object.keys(fixd).indexOf(name) >= 0) {
+      throw new Error(`Cannot add fixture to already reserved namespace '${name}'.`)
+    }
+    fixd[name] = Object.freeze(val)
+  },
+  /**
+   * Allows mutation of already existing fixture by providing name of fixture and
+   * modifier method
+   * @param {String} name The namespace of the fixture to mutate
+   * @param {Function} modifier A function to modify the fixture
+   */
+  mutate: (name, modifier) => {
```

Comment:
I like `withMutations` but seems a bit verbose, perhaps use `create`, then make the modifier optional, so the API is consistent but flexible. To visualize:

```javascript
fixd.add('foo', { fizz: 'buzz' })

fixd.foo // -> { fizz: 'buzz' }
fixd.create('foo') // -> { fizz: 'buzz' }
fixd.create('foo', () => { fizz: 'baz' }) // -> { fizz: 'baz' }
```